content/dev/autograders.md

```diff
@@ -2,4 +2,74 @@
 
 # Autograders
 
-stub
\ No newline at end of file
+PrairieLearn supports [external autograders](https://prairielearn.readthedocs.io/en/latest/externalGrading/) for assessment of students' submissions that is highly customizable. A common use case is to assess student-submitted code against tests and give a grade and feedback based on the test results.
+
+If you are working in [Python](https://prairielearn.readthedocs.io/en/latest/python-grader/), [C or C++](https://prairielearn.readthedocs.io/en/latest/c-grader/), or [Java](https://prairielearn.readthedocs.io/en/latest/java-grader/), you will likely want to use the provided autograders.
+
+If you have your own "boutique" language, someone in the PrairieLearn community may already have developed an autograder they can share with you. ([Haskell anyone](https://github.com/steven-wolfman/haskell-prairielearn)?) You can also take an existing autograder and adapt it to your needs.
```

Comment:
I have submitted a question where the Docker image is `python:3.10` and all processing is done in the question's script itself. This may be an interesting example to illustrate that it can be relatively simple to create an autograder if they have an uncommon scenario.